src/v/cluster/controller_backend.cc

```diff
@@ -570,6 +631,65 @@ ss::future<> controller_backend::fetch_deltas() {
       });
 }
 
+bool topic_files_are_orphan(
```

Comment:
The logging in this function is a bit confusing to me. `Cleaning up orphan topic files. ntp {} skip dir` makes it sound like we'll clear up.